docs/en/install-upgrade/upgrading-stack.asciidoc

```diff
@@ -1,277 +1,105 @@
+NOTE: Logstash 6.7 and Beats 6.7 are compatible with all 7.x versions of
+{es}. This provides flexibility in when you schedule the upgrades
+for your Logstash instances and Beats agents, but we recommend upgrading as
+soon as possible to take advantage of performance improvements
+and other enhancements.
 
 [[oss-stack-upgrade]]
-=== Upgrading from a pre-5.6 installation
-
-It is possible to upgrade directly to {major-version} from a pre-5.6 installation,
-but it requires a {ref}/restart-upgrade.html[full cluster restart] and you must
-manually reindex any 2.x indices you need to carry forward to {major-version}.
-
-IMPORTANT: If you use Kibana or {xpack}, you also need to upgrade the
-internal Kibana and {xpack} indices. For information about upgrading them
-after you install Elasticsearch {version}, see
-<<upgrade-internal-indices, Upgrading internal indices>>.
-
-To manually reindex a 2.x index:
-
-. Create an index with 6.x compatible mappings.
-. Use the {ref}/docs-reindex.html[reindex API] to copy documents from the
-2.x index into the new index. You can use a script to perform any necessary
-modifications to the document data and metadata during reindexing.
-. Use the {ref}/indices-aliases.html[_aliases] API to add the name of the 2.x
-index as alias for the new index and delete the 2.x index.
-
-[[upgrade-internal-indices]]
-==== Upgrading internal indices for {major-version}
-
-The format used for the internal indices used by Kibana and {xpack} has
-changed in {major-version}. Before you can run Kibana and {xpack} in {version},
-these indices must be upgraded to the new format. If you are upgrading from a
-version prior to 5.6, you must upgrade them after after installing
-Elasticsearch {version}.
-
-To get a list of the indices that need to be upgraded, install {xpack} and use
-the {ref}/migration-api-assistance.html[`_migration/assistance` API]:
-
-[source,json]
-----------------------------------------------------------
-GET /_migration/assistance
-----------------------------------------------------------
-// CONSOLE
-
-To upgrade the `.security` index:
-
-. On a single node, add a temporary superuser account to the `file` realm.
-. Use the {ref}/migration-api-upgrade.html[`_migration/upgrade`]
-API to upgrade the security index, submitting the request with the credentials
-for the temporary superuser:
-+
---
-[source,json]
-----------------------------------------------------------
-POST /_migration/upgrade/.security
-----------------------------------------------------------
-// CONSOLE
---
-
-. Delete the temporary superuser account from the file realm.
+=== Upgrading from 6.6 or earlier
 
-You can use your regular administration credentials to upgrade the other
-internal indices using the `_migration/upgrade` API.
+To upgrade directly to {version} from 6.6 or earlier, you must
+{ref}/reindex-upgrade.html[manually reindex] any 5.x indices you need to
+carry forward and perform a {ref}/restart-upgrade.html[full cluster restart].
 
```

Comment:
It might also be an option to create a new 7.x cluster and reindex from that cluster per https://www.elastic.co/guide/en/elasticsearch/reference/7.0/reindex-upgrade-remote.html ?